odoo/cli/upgrade_code.py

```diff
@@ -91,10 +90,12 @@ def content(self, value):
 class FileManager:
     addons_path: list[str]
     glob: str
+    dry_run: bool
```

Comment:
I added it because `.po` and `.pot` are not managed by the manager. An I'm kot sure if we want to add it as I don't expect code upgrades to often impact translation files.

Reply:
I think it makes sense to all all text files in `AVAILABLE_EXT`. They are not yet there because we did not have a use-case.